contrib/release/start-release.sh

```diff
@@ -18,6 +19,14 @@ usage() {
     logecho "--help     Print this help message"
 }
 
+# $1 - VERSION
+version_is_prerelease() {
+    if echo "$1" | grep -q "\(rc\)\|\(snapshot\)"; then
+        return 0
+    fi
+    return 1
+}
```

Comment:
:+1: my apologies for making review difficult here, I've tended to come at these from a rather "write-only" perspective since these scripts are byproducts of the actual release process and I'm always thinking "one of these days we'll rewrite this in Go..". Your suggestion looks sensible, I will adopt it.